src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationAttribute.kt

```diff
@@ -149,9 +156,7 @@ class LocationAttribute(
     }
   }
 
-  private fun Int.between(from: Int, to: Int) = this in from..to
-
-  private fun isPartyAllowed(party: String) = allowedParties.isNullOrBlank() || allowedParties.replace(" ", "").split(",").contains(party)
+  private fun isPartyAllowed(party: String) = allowedParties.orEmpty().replace(" ", "").split(",").contains(party)
 }
```

Comment:
`party` being either a `courtCode` or `probationTeamCode`

Reply:
Yes.  Can try and rename if not clear enough.